src/systems/breadcrumbs/Breadcrumbs.cc

```diff
@@ -160,6 +160,15 @@ void Breadcrumbs::PreUpdate(const ignition::gazebo::UpdateInfo &_info,
                 std::back_inserter(cmds));
       this->pendingCmds.clear();
     }
+    // Check that the model is valid before continuing. This check is needed
+    // because the model associated with the Breadcrumbs might have been
+    // unloaded by the level manager. Ideally, this system would have been
+    // unloaded along with the model, but that is not currently the case. See
+    // issue #113
+    if (!model.Valid(_ecm))
+    {
+      return;
```

Comment:
Do we want to print a warning in case `cmds` is not empty?

Reply:
I added a check in `Configure` in https://github.com/ignitionrobotics/ign-gazebo/pull/454/commits/9fa5c76c742c7890745bbbaee2c58c5a46773d15 that prints an error if the model is not valid. I'm not sure adding a warning here is useful because the user can't do anything about it. It is a valid use case to send the deploy command if the model is loaded back by the level manager, in which case both the original system and the new system will get the command, but the original system will silently return early here.

Reply:
Oh I see, yeah agreed that a warning would be confusing if there's a valid use case. I guess we can't tell apart when the model was removed by the `LevelManager` or by some other mechanism.